src/typhoon_automator/automator.py

```diff
@@ -164,7 +164,7 @@ def set_capture_filename(
         
         :param str filename: Path to file for capture output
         """
-        raise NotImplementedError()
+        self._capture_filename = filename
```

Comment:
Add an empty string check before this line.  This will help alert the user if an invalid capture filename has been specified.
```
if not filename:
    raise ValueError('Capture filename cannot be empty')
```
See how the 'initialize' method checks the 'schematic' parameter as another example of this